client-misk-hibernate/src/test/kotlin/app/cash/backfila/client/misk/menustack/MenuStackBackfills.kt

```diff
@@ -0,0 +1,64 @@
+package app.cash.backfila.client.misk.menustack
+
+import app.cash.backfila.client.BackfillConfig
+import app.cash.backfila.client.misk.ClientMiskService
+import app.cash.backfila.client.misk.DbMenu
+import app.cash.backfila.client.misk.MenuItem
+import app.cash.backfila.client.misk.MenuQuery
+import app.cash.backfila.client.misk.hibernate.HibernateBackfill
+import app.cash.backfila.client.misk.hibernate.SandwichParameters
+import app.cash.backfila.client.misk.hibernate.UnshardedPartitionProvider
+import app.cash.backfila.client.stat.parameters.DatasourceParameters
+import app.cash.backfila.client.stat.parameters.ParametersDatasourceBackfill
+import java.util.concurrent.BlockingDeque
+import javax.inject.Inject
+import misk.hibernate.Id
+import misk.hibernate.Query
+import misk.hibernate.Transacter
+
+class MenuStackDbBackfill @Inject constructor(
```

Comment:
```suggestion
/**
 * Backfills all menus of [SandwichParameters.type], defaulting to `beef` if unspecified.
 */
class MenuStackDbBackfill @Inject constructor(
```

Reply:
I realize now that this is actually "default to `beef` if the type parameter is explicitly set to `null`; if unspecified, type will be `chicken`" which is a bit of a mouthful!

Reply:
Yeah I was reusing a parameters class. I probably shouldn't, so I've changed this.